server/controllers/user.js

```diff
@@ -0,0 +1,55 @@
+import userRepo from '../repository/userRepository';
+import generateToken from '../utilities/jwtGenerator';
+import passwordUtil from '../utilities/passwordHasher';
+
+
+const { hash } = passwordUtil;
+/**
+ * User Controller class
+ */
+class User {
+  /**
+   * Register new User
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} User Object
+   */
+  static async signup(request, response) {
+    const {
+      firstName,
+      lastName,
+      email,
+      username,
+      password,
+    } = request.body;
+
+    const user = await userRepo.getUserByEmail(email);
+    if (user) {
```

Comment:
Can we create a formatter for our api?
It can be some sort of http utility. All you'll need to do is pass in the data and it should format it correctly. you'll probably need to pass in the response object as a parameter or something. Let me know if you have questions.

Reply:
@seyi-adeleke please this is not clear.

Reply:
Done